test/transforms/subsetFonts.js

```diff
@@ -2809,16 +2879,208 @@ describe('transforms/subsetFonts', function () {
                                 ]
                             }
                         },
+                        {
+                            type: 'HtmlNoscript',
+                            to: {
+                                type: 'Html',
+                                isInline: true,
+                                isFragment: true,
+                                outgoingRelations: [
+                                    {
+                                        type: 'HtmlStyle',
+                                        href: 'https://fonts.googleapis.com/css?family=Open+Sans'
+                                    }
+                                ]
+                            }
+                        }
+                    ]);
+                });
+        });
+    });
+
+
+    describe('with non-truetype fonts in the mix', function () {
+
+        it('should not attempt to subset non-truetype fonts', function () {
+            const warnSpy = sinon.spy().named('warn');
+
+            return new AssetGraph({root: __dirname + '/../../testdata/transforms/subsetFonts/non-truetype-font/'})
+                .on('warn', warnSpy)
```

Comment:
You can safely remove the `warnSpy`, on master the AssetGraph instance will throw if a warn event is emitted and there are no listeners.

Reply:
Removed